package.json

```diff
@@ -7,7 +7,7 @@
     "build": "./Build/createVersionFile.sh && cross-env NODE_ENV=production yarn build:ui",
     "build:ui": "NEOS_BUILD_ROOT=$(pwd) webpack --progress --colors",
     "build:ui:watch": "yarn build:ui -- --watch",
-    "build:ui:watch-poll": "yarn run build:ui --watch-poll --watch",
+    "build:ui:watch-poll": "yarn run build:ui -- --watch-poll --watch",
```

Comment:
What about this?
`warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.`
Also for me, the resulting command is exactly the same, do you have an old yarn version?

Reply:
The script just exits for me after the first run without it...

Reply:
Oh wait, I do! Will fix.

Reply:
`0.27.5` seems to be the latest version available for Ubuntu... Could we maybe leave this like that for now, and in some time remove `--` from both commands? Otherwise somebody on lower versions of yarn could also waste some time on this...